CHANGELOG.txt

```diff
@@ -12,6 +12,7 @@
 * Fixed a bug that did not allow expected input in Standard I/O type question to be none.
 * UI changes in grade user, view answerpaper and monitor pages.
 * Fixed a bug that would require shebang to be put in for Bash assertion based questions.
+* Latest Course will be shown on top of Courses Page.
```

Comment:
For continuity's sake , change it to past tense. Also change it from latest courses to active courses.

Reply:
Also, do the same for the point on line no 13.. i.e. -
**UI changes in grade user, view answerpaper and monitor pages** should be **Changed UI in grade user, view answerpaper and monitor pages**.